### ISSUE TITLE
initial LP solve does not obey timelimit

### ISSUE THREAD

**Issue by svigerske** [coin-or/Cbc#231]
If I run http://miplib2010.zib.de/download/ns1758913.mps.gz with a short timelimit, Cbc does not stop before it finished solving the first LP:
```
Welcome to the CBC MILP Solver 
Version: Trunk (unstable) 
Build Date: May 31 2019 
Revision Number: 2580 

command line - ./bin/cbc -import ns1758913.mps.gz -seconds 10 -branch (default strategy 1)
At line 1 NAME           ns1758913
At line 2 ROWS
At line 624170 COLUMNS
At line 1907619 RHS
At line 2513834 BOUNDS
At line 2531791 ENDATA
Problem ns1758913 has 624166 rows, 17956 columns and 1283444 elements
Coin0008I ns1758913 read with 0 errors
seconds was changed from 1e+100 to 10
Continuous objective value is -1501.18 - 53.01 seconds
Cbc0020I Exiting on maximum time
Cbc0005I Partial search - best objective 1e+50 (best possible -1501.1833), took 0 iterations and 0 nodes (54.90 seconds)
Cbc0035I Maximum depth 0, 0 variables fixed on reduced cost
Cuts at root node changed objective from -1501.18 to -1501.18
Probing was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
Gomory was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
Knapsack was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
Clique was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
MixedIntegerRounding2 was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
FlowCover was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
TwoMirCuts was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)
ZeroHalf was tried 0 times and created 0 cuts of which 0 were active after adding rounds of cuts (0.000 seconds)

Result - Stopped on time limit

No feasible solution found
Lower bound:                    -1501.183
Enumerated nodes:               0
Total iterations:               0
Time (CPU seconds):             54.91
Time (Wallclock seconds):       56.00

Total time (CPU seconds):       55.87   (Wallclock seconds):       57.04
```
This issue seems very similar to #168, which is about ignoring the timelimit in the postsolve-LP.

**Comment by h-g-s** [comment]
OsiSolverInterface should have a timeLimit,  now there is only iterationLimit. Time limit is quite important!

**Comment by svigerske** [comment]
The Osi core developers refused to add timelimits for a long while. There were some fruitless discussions.

But @jjhforrest  seems to have fixes this with fe2bc468. Thanks.
